terraform/deployments/apps/content-store-tmp/live.tf

```diff
@@ -0,0 +1,46 @@
+module "live_container_definition" {
+  source = "../../../modules/app-container-definition"
+  name   = "live-content-store"
+  image  = "govuk/content-store:bill-content-schemas" # TODO use "govuk/content-store:${var.image_tag}"
+  environment_variables = merge(
+    local.environment_variables,
+    {
+      PLEK_SERVICE_ROUTER_API_URI = "http://router-api.${local.mesh_domain}"
+      MONGODB_URI                 = "mongodb://${local.mongodb_host}/live_content_store_production"
+    },
+  )
+  log_group             = local.log_group
+  secrets_from_arns     = local.secrets_from_arns
+  aws_region            = data.aws_region.current.name
+  depends_on_containers = { envoy : "START" }
+}
+
+module "live_envoy_configuration" {
+  source = "../../../modules/envoy-configuration"
+
+  mesh_name    = local.mesh_name
+  service_name = "live-content-store"
+  log_group    = local.log_group
+  aws_region   = data.aws_region.current.name
+}
+
+resource "aws_ecs_task_definition" "live" {
+  family                   = "content-store"
```

Comment:
Perhaps this should be `live-content-store`, since we have the other family as `draft-content-store` and that name scheme used throughout.

Reply:
Would we then put the `live-` prefix on everything though? What about services which don't have a draft version? Would that make it clearer or more obscure for someone encountering this for the first time?

Reply:
I think this might be a case where the least surprising thing to do is to keep using the same names as before, e.g. plain content store is "content-store" and draft is "draft-content-store". What do you think?

Reply:
Yeah I agree.